_security/multi-tenancy/mt-agg-view.md

```diff
@@ -36,7 +36,10 @@ In this first experimental phase of development, there are some limitations that
 
 * The feature can only be used in a new cluster. At this time, the feature is not suported by clusters already in use.
 * Also, the feature should be used only in a test environment, not in production. 
-* Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and have an impact on tenant-to-tenant functionality. 
+* Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and can have an impact on tenant-to-tenant functionality.
+
+  While aggregate view is enabled, a loss of saved objects and impact to tenant-to-tenant functionality can also occur when the larger multi-tenancy feature is disabled. This includes disabling multi-tenancy in the security `config.yml` file settings or by using dynamic configuration. Keep this in mind before enabling aggregate view for your test cluster.
```

Comment:
@RyanL1997 I don't think there will be a loss of saved objects with respect to Aggregation view feature when multi-tenancy is disabled. Can you confirm?

Reply:
@abhivka7 @RyanL1997 I understood that some changes needed to be made so that aggregate view was compatible with dynamic configuration. The result of those changes was that disabling multi-tenancy would also disable the aggregate view feature. And once both were disabled, it made the last bullet above a concern:
"Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. ..."
Let's clarify this so I can make changes to the warning note if necessary. Thanks.

Reply:
@cwillum Basically aggregation view is only shown when multitenancy is enabled. If we disable multitenancy then there is no point in showing aggregation view. So aggregation view as a feature in itself can not be disabled once it has been enabled. But it still needs multitenancy to be enabled as a pre-requisite.

Reply:
Hi @abhivka7 and @cwillum . I will just move some of our conversation over here for references:
1. This potential risk / known limitation existed since the launch of the aggregation view feature, referencing to the previous documentation here (https://opensearch.org/docs/latest/security/multi-tenancy/mt-agg-view/#known-limitations): 
> Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and have an impact on tenant-to-tenant functionality.
2. According to the new feature of Dynamically Configurable Multi-tenancy, it didn't change any previous technical design of both aggregation view feature and tenancy feature. However, it does change some accessibility of multi-tenancy configuration. That's reason of @cwillum  is helping us to summarizing and emphasizing some specific details of these potential user behaviors, so that we can warn user about the above risks even with a non-production environment

All in all, big thanks to both of you for the follow-ups. I think we have sorted everything clear.